openfisca_uk/situation_examples/examples.py

```diff
@@ -0,0 +1,71 @@
+def single_person_UC(sim):
+    sim.add_person(
+        name="adult", age=26, is_household_head=True, is_benunit_head=True
+    )
+    sim.add_benunit(adults=["adult"], claims_UC=True)
+    sim.add_household(adults=["adult"])
+    return sim
+
+
+def couple_UC(sim):
+    sim.add_person(
```

Comment:
Right now actually no, because there's no simple function for changing values outside of editing the internal JSON, and the group variables would need to have the extra person added.